### ISSUE TITLE
Do we have to use validation sets？

### ISSUE THREAD

**Issue by choresefree** [UCSC-REAL/cifar-10-100n#2]
Dear game initiator, I noticed that the official code divides the verification set. Do we also need to divide the verification set? Can we train on the complete training set and then test on the test set？

**Comment by zwzhu-d** [comment]
Hi [choresefree](https://github.com/choresefree),

We need to select the best model on noisy validation data as the one for the final evaluation on test data. If you trust the stability of your model, you can train with all the training data but we will use the **last epoch** model if the training pipeline does not have a validation step. 

Best,
Zhaowei

**Comment by zwzhu-d** [comment]
> Dear game initiator, I noticed that the official code divides the verification set. Do we also need to divide the verification set? Can we train on the complete training set and then test on the test set？

Hi [choresefree](https://github.com/choresefree),

We need to select the best model on noisy validation data as the one for the final evaluation on test data. If you trust the stability of your model, you can train with all the training data but we will use the **last epoch** model if the training pipeline does not have a validation step. 

Best,
Zhaowei

**Comment by choresefree** [comment]
> > Dear game initiator, I noticed that the official code divides the verification set. Do we also need to divide the verification set? Can we train on the complete training set and then test on the test set？
> 
> Hi [choresefree](https://github.com/choresefree),
> 
> We need to select the best model on noisy validation data as the one for the final evaluation on test data. If you trust the stability of your model, you can train with all the training data but we will use the **last epoch** model if the training pipeline does not have a validation step.
> 
> Best, Zhaowei

Thank you very much for your reply,
